### DISCUSSION TITLE
Github actions organization self-hosted runners edless loading

### DISCUSSION THREAD

**Discussion by io112** [community/community#26458] [Actions]
<p>Cant’t add self-hosted runner to organization. Endless loading.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/3c2e6a3e68720714255079276ceb5201b8c9645a.png" data-download-href="/uploads/short-url/8Ao5HpgMQEHHv34QUQtnljYlT06.png?dl=1" title="Annotation 2020-04-25 144427.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/17797416/181097897-f14a0ad5-dfb4-40ed-9bc3-e518b215df9b.png" alt="Annotation 2020-04-25 144427.png" data-base62-sha1="8Ao5HpgMQEHHv34QUQtnljYlT06" width="690" height="287" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/3c2e6a3e68720714255079276ceb5201b8c9645a_2_690x287.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/3c2e6a3e68720714255079276ceb5201b8c9645a_2_1035x430.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/3/3c2e6a3e68720714255079276ceb5201b8c9645a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/3c2e6a3e68720714255079276ceb5201b8c9645a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Annotation 2020-04-25 144427.png</span><span class="informations">1313×547 35.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mscoutermarsh** [comment]
<p>Hi <a class="mention" href="/io112">@io112</a>,</p>
<p>Is this still happening? Can you share the organization where it’s occuring? or email me mscoutermarsh AT github.</p>
<p>Thanks! </p>

**Comment by io112** [comment]
<p>Yep, here it is <a href="https://github.com/IT108" rel="nofollow noopener">https://github.com/IT108</a>.</p>

**Comment by mscoutermarsh** [comment]
<p>Thanks, will get someone to look into this.</p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/io112">@io112</a> ,</p>
<p>Just to confirm is org level self-hosted runner ok for you now? If not, it’s recommended to ask private support here: <a href="https://support.github.com/contact" rel="nofollow noopener">https://support.github.com/contact</a></p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Hi <a class="mention" href="/io112">@io112</a>,</p>
<p>I looked into this and found the problem.</p>
<p>A temporary work around you can try: In any repository in your org, if you trigger a hosted Actions run (could be a blank workflow). Then the self-hosted page should start working again.</p>
<p>We’re working on a permanent fix for this.</p>
<p>By the way, is it possible you recently transferred any repositories into this Org?</p>
<p>Thanks!</p>

**Comment by io112** [comment]
<p>Thanks, i already solved problem by adding self-hosted runner in one repository and delete it.</p>
